parm/atm/berror/hybvar_bump.yaml.j2

```diff
@@ -105,7 +105,7 @@ components:
         filename_sfcw: '{{ current_cycle | to_fv3time }}.fv_srf_wnd.res.nc'
         filename_cplr: '{{ current_cycle | to_fv3time }}.coupler.res'
       pattern: '%mem%'
-      nmembers: $(NMEM_ENS)
+      nmembers: '{{ NMEM_ENS }}'
```

Comment:
This should not be quoted. Quoting will make it a string, and this is an integer.

Reply:
Quotes removed at e3cdbcf